Projects/Exadel.ReportHub/Exadel.ReportHub.Host/Filters/GlobalExceptionFilter.cs

```
@@ -1,6 +1,31 @@
-﻿namespace Exadel.ReportHub.Host.Filters
+﻿using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Exadel.ReportHub.Host.Filters
 {
-    public class GlobalExceptionFilter
+    public class GlobalExceptionFilter : IExceptionFilter
     {
+        public void OnException(ExceptionContext context)
+        {
+            var statusCode = context.Exception switch
```

just catch all exceptions Exception and set HttpStatusCode.InternalServerError always, no need to do this switch, other status codes will be handled separately, not with exceptions handling